apps/guardian-ui/src/components/SetConfiguration.tsx

```diff
@@ -279,6 +282,22 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
                 : t('set-config.admin-password-set')}
             </Text>
           </FormHelperText>
+          {password && (
+            <FormControl>
+              <FormLabel>{t('set-config.confirm-password')}</FormLabel>
+              <Input
+                type='password'
+                value={confirmPassword}
+                onChange={(ev) => setConfirmPassword(ev.currentTarget.value)}
+                placeholder='Confirm Password'
+              />
+              {password !== confirmPassword && (
+                <FormHelperText color='red'>
+                  {t('set-config.error-password-mismatch')}
+                </FormHelperText>
+              )}
+            </FormControl>
+          )}
```

Comment:
Wait, if this is on the same screen won't people just copy/paste it? Should probably be a "surprise" when they try to click continue.

Reply:
kk I can do that give me a sec

Reply:
<img width="662" alt="image" src="https://github.com/fedimint/ui/assets/74332828/d076c8ae-cfd1-41ed-97c9-8524b4cb2b66">

Confirm password pops as a modal when you click next then you have to verify it